ingesters/winevents/main.go

```diff
@@ -74,27 +72,34 @@ func main() {
 		fmt.Fprintf(os.Stderr, "Failed to get interactive session status: %v\n", err)
 		return
 	}
-	if !inter {
+	if inter {
+		lg = log.New(os.Stdout)
+	} else {
+		lg = log.NewDiscardLogger()
 		e, err := eventlog.Open(serviceName)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to get event log handle: %v\n", err)
+			lg.Error("failed to get event log handle", log.KVErr(err))
```

Comment:
We've just set lg to a NewDiscardLogger, so this message is guaranteed to be thrown away every time, right? I think printing to stderr is a better idea.

Reply:
agreed, fix incoming